contentpacks/utils.py

```diff
@@ -704,3 +704,14 @@ def remove_nonexistent_assessment_items_from_exercises(node_data: list, assessme
             except Exception as e:
                 import pdb; pdb.set_trace()
                 print(1)
+
+def remove_exercises_without_assessment_items(node_data):
```

Comment:
You can just add a `TODO` mark to remind us to refactor these functions in the future.